content/docs/dvclive/api-reference/env.md

```diff
@@ -0,0 +1,17 @@
+# dvclive.env
```

Comment:
Not a blocker: should this be inside the api ref or outside of it?

Reply:
I am more inclined towards leaving it under api-reference

Reply:
Perhaps when adding the Studio live metrics variables it has enough value to move outside, but I kind of like the current structure with only 2 pages + API ref and technically I consider the env vars to belong inside the API Ref